Sources/MapboxSearch/PublicAPI/Use Cases/Discover API/Discover.swift

```diff
@@ -59,9 +59,11 @@ extension Discover {
         userActivityReporter.reportActivity(forComponent: "discover-search-nearby")
 
         let searchOptions = SearchOptions(
+            countries: [options.country?.countryCode].compactMap { $0 },
             languages: [options.language.languageCode],
             limit: options.limit,
-            proximity: proximity
+            proximity: options.proximity ?? proximity,
```

Comment:
As for me, here it should be rather
```suggestion
            proximity: proximity
```
This method takes non-optional argument, so it should be prioritized over `options.proximity`

Reply:
Yeah you're right to use the non-optional with priority.